test/ref/pack_int4.cpp

```diff
@@ -91,10 +107,8 @@ TEST_CASE(pack_int4_axis_0)
     EXPECT(migraphx::verify::verify_rms_range(results_vector, gold));
 }
 
-TEST_CASE(pack_int4_nchw)
+TEST_CASE(pack_uint4_nchw)
 {
-    // test with literal values such as 0x18 in which first 4 bits will be dropped, ideally
-    // quantizer should produce values that fit into 4 bits.
```

Comment:
I would change this comment to: `test with literal values such as 0x18 in which first 4 bits will be dropped. The pack should clip these values`.

Reply:
Comments have been updated, though not verbatim. Thanks